services/vpc_manager/src/test/java/com/futurewei/alcor/vpcmanager/SegmentControllerTests.java

```diff
@@ -46,6 +48,9 @@
     @MockBean
     private SegmentDatabaseService segmentDatabaseService;
 
+    @MockBean
+    private VpcDatabaseService vpcDatabaseService;
```

Comment:
I think we will need one more test case to validate that if all ranges are allocated and the new codes could throw out the keyNotEnough exception.

Reply:
got it